loadtest/main.go

```diff
@@ -110,48 +111,89 @@ func generateMessage(config Config, key cryptotypes.PrivKey, msgPerTx uint64, va
 			panic("Unknown message type")
 		}
 	case "dex":
-		msgType := config.MsgTypeDistr.SampleDexMsgs()
-		orderPlacements := []*dextypes.Order{}
-		var orderType dextypes.OrderType
-		if msgType == "limit" {
-			orderType = dextypes.OrderType_LIMIT
-		} else {
-			orderType = dextypes.OrderType_MARKET
-		}
-		var direction dextypes.PositionDirection
-		if rand.Float64() < 0.5 {
-			direction = dextypes.PositionDirection_LONG
-		} else {
-			direction = dextypes.PositionDirection_SHORT
-		}
-		price := config.PriceDistr.Sample()
-		quantity := config.QuantityDistr.Sample()
-		contract := config.ContractDistr.Sample()
-		for j := 0; j < int(msgPerTx); j++ {
-			orderPlacements = append(orderPlacements, &dextypes.Order{
-				Account:           sdk.AccAddress(key.PubKey().Address()).String(),
-				ContractAddr:      contract,
-				PositionDirection: direction,
-				Price:             price.Quo(FromMili),
-				Quantity:          quantity.Quo(FromMili),
-				PriceDenom:        "SEI",
-				AssetDenom:        "ATOM",
-				OrderType:         orderType,
-				Data:              VortexData,
-			})
-		}
-		amount, err := sdk.ParseCoinsNormalized(fmt.Sprintf("%d%s", price.Mul(quantity).Ceil().RoundInt64(), "usei"))
-		if err != nil {
-			panic(err)
-		}
-		msg = &dextypes.MsgPlaceOrders{
-			Creator:      sdk.AccAddress(key.PubKey().Address()).String(),
-			Orders:       orderPlacements,
-			ContractAddr: contract,
-			Funds:        amount,
+		msgType := c.LoadTestConfig.MsgTypeDistr.SampleDexMsgs()
```

Comment:
can you rebase your loadtest script onto `cherry-pick-parallel` instead of 2.0.0beta? theres several changes made there that I'd prefer get included by default instead of rebasing conflicts

Reply:
Im getting a lot of merge conflicts when trying to rebase it on loadtest/main.go, maybe i can just revert the loadtest/changes to 2.0.0beta and make the loadtest/changes against your branch in another PR?